include/fw_metadata.h

```diff
@@ -24,9 +24,11 @@
 #include <stddef.h>
 #include <toolchain.h>
 #include <sys/util.h>
-#include <assert.h>
+#include <sys/__assert.h>
```

Comment:
Why is this changed?

Reply:
yes, something changed, and then we got undefined errors.